generated/google_cloud_protobuf/lib/protobuf.dart

```diff
@@ -22,6 +22,8 @@ library;
 import 'package:google_cloud_gax/gax.dart';
 import 'package:google_cloud_gax/src/encoding.dart';
 
+export 'package:google_cloud_gax/gax.dart' show Any;
```

Comment:
As this file is generated, you'll need to introduce a way to define exports in a sidekick file.